users-api/api/put-user.spec.js

```diff
@@ -0,0 +1,18 @@
+
+const app = require('../app.js');
+const should = require('should');
+const mocha = require('mocha');
+const supertest = require('supertest');
+
+describe('/users/:id', () => {
+  it('should update a users information', (done) => {
+    supertest(app)
+    .put('/users/:id')
+    .end(err, res) => {
+      res.json.should.equal({
```

Comment:
Does this pass when you run `npm test` from inside the `users-api` folder? I think it should be:

```javascript
res.body.should.equal(':id');
```

or something of the sort